lib/tests/streamlit/watcher/event_based_path_watcher_test.py

```diff
@@ -99,12 +96,48 @@ def test_works_with_directories(self):
 
         ro.close()
 
+    def test_kwargs_plumbed_to_calc_md5(self):
```

Comment:
Could you add a docstring to the test? (What kwargs are being plumbed, and why?)

Reply:
👍 done